src/Blocks/FormControl.purs

```diff
@@ -50,8 +51,8 @@ formControl
 formControl props html =
   HH.div
     [ HP.classes formControlClasses ]
-    [ HH.span
-      [ HP.class_ (HH.ClassName "w-full") ]
+    [ HH.label
```

Comment:
This is still wrapping everything in a label. We'll need to move the nesting out:

```purescript
[ HH.label
  [ HP.class_ (HH.ClassName "w-full"), HP.for props.inputId ]
  [ label props.label ]
, HH.div
  [ HP.class_ (HH.ClassName "my-1") ]
  [ html ]
, helpText props.valid props.helpText
]
```

Reply:
Yeah I think we should do that, however it appears to be working for me. It's possible that including the `for` overrides the implicit association with all of it's children.